src/main/webapp/app/overview/exercise-details/course-exercise-details.component.html

```diff
@@ -119,7 +119,7 @@
         <div class="col pe-0 flex-grow-1">
             <!-- EXERCISE ACTIONS END -->
             <!-- RESULTS START -->
-            <div class="row" *ngIf="this.sortedHistoryResults?.length && this.sortedHistoryResults.length > 1">
+            <div class="row" *ngIf="(this.sortedHistoryResults?.length && this.sortedHistoryResults.length > 1) || this.practiceStudentParticipation?.results?.length">
```

Comment:
would prefer calling a function in the component rather for more complex conditions like this

Reply:
```suggestion
            <div class="row" *ngIf="!!this.sortedHistoryResults?.length || this.practiceStudentParticipation?.results?.length">
```
I think this could be simplified to this as `>0` is basically a "not a falsy value".
Correct me if I'm wrong.
Our policy is not to use function calls there to prevent constant recalculations.

Reply:
As Julian already said, we try to avoid functions in html, since they are sometimes invoked multiple times per second if the change detection is not working properly.
And @julian-christl your refactoring would be correct for `>0` but we have `>1` here.